src/function/table/table_scan.cpp

```diff
@@ -769,6 +769,8 @@ InsertionOrderPreservingMap<string> TableScanToString(TableFunctionToStringInput
 	InsertionOrderPreservingMap<string> result;
 	auto &bind_data = input.bind_data->Cast<TableScanBindData>();
 	result["Table"] = bind_data.table.name;
+	result["Schema"] = bind_data.table.schema.name;
+	result["Catalog"] = bind_data.table.schema.catalog.GetName();
```

Comment:
Can we do this instead?
```C++
result["Table"] = ParseInfo::QualifierToString(bind_data.table.schema.catalog.GetName(), bind_data.table.schema, bind_data.table.name);
```
Then we have all the information in one field, and empty/invalid schema/catalog are handled nicely.